ccd-client/src/test/java/uk/gov/hmcts/cmc/ccd/client/service/StartCaseTest.java

```diff
@@ -0,0 +1,82 @@
+package uk.gov.hmcts.cmc.ccd.client.service;
+
+import com.sun.tools.javadoc.Start;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.http.*;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.hmcts.cmc.ccd.client.header.HttpHeadersFactory;
+import uk.gov.hmcts.cmc.ccd.client.model.EventRequestData;
+import uk.gov.hmcts.cmc.ccd.client.model.StartEventResponse;
+
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.eq;
+
+@RunWith(MockitoJUnitRunner.class)
+public class StartCaseTest {
+
+    @Mock
+    private RestTemplate restTemplate;
+    @Mock
+    private HttpHeadersFactory headersFactory;
+
+    private StartCase startCase;
+
+    private EventRequestData eventRequestData = new EventRequestData();
+
+    @Mock
+    private HttpHeaders httpHeaders;
+
+    @Before
+    public void setup() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test
+    public void testExchangeReturnsStartEvent() throws Exception {
+        when(headersFactory.getHttpHeader()).thenReturn(httpHeaders);
+        ResponseEntity<StartEventResponse> responseEntity = Mockito.mock(ResponseEntity.class);
+        HttpEntity<String> httpEntity = Mockito.mock(HttpEntity.class);
+
+        when(restTemplate.exchange(Mockito.anyString(),
+            eq(HttpMethod.GET),
+            eq(httpEntity),
+            eq(StartEventResponse.class) )).thenReturn(responseEntity);
+
+        StartEventResponse startEventResponse = startCase.exchange(eventRequestData);
+
+        Assert.assertNotNull(startEventResponse);
+
+        Mockito.verify(headersFactory).getHttpHeader();
+        Mockito.verify(restTemplate).exchange(Mockito.anyString(),
```

Comment:
Please use static import of `verify` method instead of referencing the class a number of times.